### PR TITLE
Adding new goal to remove image with or without run configuration

### PR DESCRIPTION
**Pull request by nirkoren**
Hi, This is very important for us to remove image on any case with or without a run configuration

### CODE REVIEW

**Comment by rhuss** [comment]
thanks ! just coming back from a conf, will have a look soon.

**Comment by nirkoren** [comment]
Hi,
Done. I unified both. The removeAll deleting only data images. I need some agnostic step to remove any kind of images. 
Thanks,
Nir

**Comment by rhuss** [comment]
`removeAll` removed every existing image, regardless whether its a data image or not --> https://github.com/fabric8io/docker-maven-plugin/pull/553/files#diff-fdcc75a388b6c25b3a161fe9f4d1059dL56

Still dont see the advantage of `cleanup` over the old `removeAll` option.

File: src/main/java/io/fabric8/maven/docker/RemoveMojo.java [hunk 1]
```diff
@@ -43,23 +45,34 @@
...
+				if (queryService.hasImage(name)) {
+					deleteImage(hub, image, name);
+				}
+			}
+		}
+	}
 
-        for (ImageConfiguration image : getResolvedImages()) {
-            String name = image.getName();
-            if (removeAll || image.isDataImage()) {
```
Comment (rhuss) [inline src/main/java/io/fabric8/maven/docker/RemoveMojo.java]: remove all or remove when its a data image

**Comment by nirkoren** [comment]
I am closing this PR and will open a new one with a small change

**Pull request closed without merging**